Automattic/wp-calypso
# Refactor sites per plugin view to use DataViews

Fixes https://github.com/Automattic/dotcom-forge/issues/8081

## Proposed Changes

* Update the `SitesWithInstalledPluginsList` component to render a table with DataViews
* Remove the existing `SiteList` component

## Why are these changes being made?


* https://github.com/Automattic/dotcom-forge/issues/8081

## Testing Instructions



* Go to `/plugins/manage` and click a plugin
* Check on the fly-out panel that the `Installed On` table is still rendered correctly
* Test the per-site actions, and check if everything still works as before


| Trunk | This branch |
|--------|--------|
| ![CleanShot 2025-01-03 at 15 23 09@2x](https://github.com/user-attachments/assets/b9ae5f8b-d248-4dbe-9c24-0bc0cb8e24da) | ![CleanShot 2025-01-03 at 15 22 55@2x](https://github.com/user-attachments/assets/e7421e86-c3d6-4315-a318-f79988852e94) | 

Plugin actions:

https://github.com/user-attachments/assets/fd501ecb-f806-4cab-9828-47bf7cf93e3d
